include/crow/http_connection.h

```diff
@@ -418,6 +418,7 @@ namespace crow
                 return;
             }
 
+            // Keep in sync with common.h/status
             static std::unordered_map<int, std::string> statusCodes = {
                 {100, "HTTP/1.1 100 Continue\r\n"},
```

Comment:
Maybe change status members to enum entries?

Reply:
Could you elaborate please?

Reply:
Instead of mapping an `int` to a `string`, he suggests directly mapping the enum type `status` to a `string`.

Reply:
hmmmm, I'll test that approach

Reply:
done, all tests ran fine and anyone can still use integers or the enum as the response code.